### PR TITLE
chore: tags

### PR DESCRIPTION
**Pull request by Syjalo**
A new tag for interaction replies

### CODE REVIEW

File: tags/tags.toml [hunk 1]
```diff
@@ -895,3 +895,12 @@ content = """
 + embed.setFooter({ text: 'This is an example text', iconURL: 'https://example.com/icon.png' })
 ```
 """
+
+[interaction-acknowledged]
```
Comment (almostSouji) [inline tags/tags.toml:899]:
```suggestion
[already-acknowledged]
```
already-acknowledged is the essence of the error

File: tags/tags.toml [hunk 2]
```diff
@@ -895,3 +895,12 @@ content = """
...
+keywords = ["interaction-acknowledged", "interaction", "acknowledged", "reply", "deferReply"]
```
Comment (almostSouji) [inline tags/tags.toml:900]:
```suggestion
keywords = ["already-acknowledged", "interaction-acknowledged"]
```
keywords need to be unique, some of these were way too generic

File: tags/tags.toml [hunk 3]
```diff
@@ -895,3 +895,12 @@ content = """
...
+content = """
+`DiscordAPIError: Interaction has already been acknowledged`
+You have alredy replied to the interaction by `<Interaction>.reply()` or `<Interaction>.deferReply()`.
```
Comment (almostSouji) [inline tags/tags.toml:903]: These are not the only possible initial responses (modals coming up as well), unsure if it's wise to list them all, but if you decide to do so, list all, I guess

File: tags/tags.toml [hunk 4]
```diff
@@ -895,3 +895,12 @@ content = """
...
+• Use `<Interaction>.followUp()` to send a new message
+• If you defered reply it's better to use `<Interaction>.editReply()`
```
Comment (almostSouji) [inline tags/tags.toml:905]: deferred (spelling)
why is it "better"? implies that the current thing is also an option, which it really isn't

**Review by almostSouji — CHANGES_REQUESTED** [review]
- Keywords too generic
- Spelling
- Link to guide as in other tags (link md syntax)

File: tags/tags.toml (hunk 4 above)
Comment (Syjalo) [reply tags/tags.toml]: So it should be like this?
If you deferred reply use `<Interaction>.editReply()`

**Review by iCrawl — APPROVED** [review]

**Review by almostSouji — APPROVED** [review]

**Pull request merged**